dev-docs/bidders/iionads.md

```diff
@@ -22,11 +22,22 @@ sidebarType: 1
 
 ### Bid Params
 
+#### Prebid.js
+
+{: .table .table-bordered .table-striped }
+
+| Name          | Scope    | Description                                       | Example         | type      |
+|:--------------|:---------|:--------------------------------------------------|:----------------|:----------|
+| `host`        | required | Ad network's RTB host                             | `'iionads.com'` | `string`  |
+| `adUnitId`    | required | Ad Unit Id will be generated on iionads Platform. | `42`            | `integer` |
```

Comment:
This breaks our rule that the parameters for client-side and server-side adapters are the same. It appears that your server-side adapter doesn't support either adUnitId or adUnitType. This makes it awkward for publishers who want to use the Server-to-Server test module to try X% of their traffic client-vs-server.

I would like you to update your server-side adapter to accept adUnitId and adUnitType and just ignore them if you don't need them there for some reason.

Reply:
@bretg Hi, thank you for your review. You are exactly right. I think I have fixed it. Please check it again.